### PR TITLE
add swiftjsonmodeler cask

### PR DESCRIPTION
**Pull request by CodeOcenS**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/swiftjsonmodeler.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask "swiftjsonmodeler" do
```
Comment (vitorgalvao) [inline Casks/swiftjsonmodeler.rb:1]: `swiftjsonmodeler-for-xcode`

File: Casks/swiftjsonmodeler.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  version "2.0.0"
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/swiftjsonmodeler.rb:3]: `sha256` also needs to be set. You can find what it is by running `shasum --algorithm 256` on the downloaded file.

File: Casks/swiftjsonmodeler.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+  url "https://github.com/CodeOcenS/SwiftJSONModeler/releases/download/v#{version}/SwiftJSONModeler.For.Xcode.app.zip"
+  appcast "https://github.com/CodeOcenS/SwiftJSONModeler/releases.atom"
+  name "SwiftJSONModeler"
+  desc "The Swift JSON Model generator for Xcode extension(Swift JSON 转模 Xcode 插件)"
```
Comment (vitorgalvao) [inline Casks/swiftjsonmodeler.rb:8]: Remove the parenthetical.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Thank you, but we’ll be rejecting this on account of [not being notable enough](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):

> + The Cask is too obscure. Examples:
>   + A self-submitted app from a GitHub repository that is [not notable enough](https://github.com/Homebrew/homebrew-cask/pull/28103) (under 30 forks, 30 watchers, and 75 stars).
>   + [Electronic Identification (eID) software](https://github.com/Homebrew/homebrew-cask/issues/59021).

**Comment by CodeOcenS** [comment]
OK, I understand. Thank you.

**Pull request closed without merging**